frt-part-two.md

```diff
@@ -0,0 +1,55 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | The Face of Tomorrow Part Two |
+| Headline  | FRT where innovation meets controversy. Are you concerned about privacy? |
+| Cypherpunktimes | cypherpunktimes.com/...link |
+| Publish Date | 00 Feb 2024 |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement - FRT laws focus on giving power to enforcement at the expense of personal freedoms and privacies. Is this the technology that facilitates an Orwellian future or is this the innovation that propels humanity to greater heights?
+
+Pre-discussion, research and questions
+
+
+# The Face of Tomorrow Part Two
+
+First draft.... need to add links and use cases and more links
+
+
+Introduction:
```

Comment:
Markdown formatting. For subtitles use two hash tags followed by a space and a line break under the title. This will make the title larger. Markdown can also be copied directly into CT, all formatting should present perfectly.


Example (ignore —):

— ## Introduction

this is my intro

Reply:
Thanks I have done this now. Pity that the site https://www.markdownguide.org/basic-syntax/ seems to be down just now when I visited